ocs_ci/deployment/vmware.py

```diff
@@ -544,6 +566,15 @@ def deploy_ocp(self, log_cli_level='DEBUG'):
                 (default: "DEBUG")
 
         """
+        cluster_name_parts = config.ENV_DATA.get("cluster_name").split("-")
+        prefix = cluster_name_parts[0]
+        if not prefix.startswith('jnk'):
```

Comment:
Thanks @ebenahar , my point was also to use this prefix from constants.
So please define:
PRODUCTION_JOBS_PREFIX in https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py
and use it here.
Later on we will change on one place when we will need to change the prefix.

Reply:
done